templates/core/search_results.html

```diff
@@ -10,9 +10,13 @@
         ></span>
         <img class="album-cover" src="{{ album_info.cover.url }}" />
 
-        <div class="album-info">
-          <div class="album-title">{{ album_info.name }}</div>
-          <div class="artist">{{ album_info.artist }}</div>
+        <div class="album-info" data-uri="{{album_info.album_uri}}">
```

Comment:
I noticed a bug where if you click the plus icon instead of the album info div to add the album to the rec list, we get a TypeError when we try to get the dataset from that element. This is because the target element is different. So my suggestion to fix this is to move the data-uri up to the div that is the container for all these elements, including the span containing the icon, and use that instead of `album-info` in our dom selection methods. I'll add a comment on that JS file too.